src/desktop/apps/notifications/stylesheets/index.styl

```diff
@@ -13,51 +14,10 @@
   .filter-artist-forsale-artworks-count
     display none
 
-#notifications-page[data-state=loading]
```

Comment:
All these can be removed because the React app handles the display of these works, at least on desktop.